include/matx/transforms/svd/svd_cuda.h

```diff
@@ -580,6 +580,10 @@ static __MATX_INLINE__ SVDMethod GetCUDASVDMethod(const ATensor &a) {
   return method;
 }
 
+#ifndef DOXYGEN_ONLY
+// Hide this from doxygen, otherwise we get
```

Comment:
When I hit these errors usually either updating doxygen or using the Sphinx equivalent parser fixed it.